store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -104,7 +104,8 @@ public final class VolumeCLI extends SoapCLI {
     private static final String A_BUCKET_ID = "bucketId";
     private static final String A_VOLUME_PREFIX = "volumePrefix";
     private static final String A_VOLUME_S3 = "S3";
-    
+    private static final String A_VOLUME_OPEN_IO = "OPENIO";
```

Comment:
Check if we have a constant for it and use that and make sure this is the same value that we are accepting creating volume.

Reply:
Used Admin constants for A_VOLUME_OPEN_IO and A_VOLUME_S3